CONTRIBUTING.md

```diff
@@ -62,30 +62,31 @@ questions: what changed and why. The subject line should feature the what and
 the body of the commit should describe the why.
 
 ```
-capture: add comment for variable declaration
+capture(ticdc): add comment for variable declaration
 
 Improve documentation.
 ```
 
 The format can be described more formally as follows:
 
 ```
-<subsystem>: <what changed>
+<subsystem>(ticdc|dm|both): <what changed>
```

Comment:
Recommend to use explicit subsystem name(s), for two reasons
- There may be other subsystem
- Explicit name(s) is good for searching

Reply:
Sorry, I don't quite understand what you mean. Do you mean putting the prefix in front?
`ticdc/xxxx,dm/xxxx: xxxx`?

Reply:
I mean write all subsystems explicitly, such as 
- `ticdc,dm/xxx`, if affects both ticdc and dm
- `ticdc,dm,something-else,another-subsystem/xxx` it affects more than two subsystems.
- `ticdc/xxx` or `dm/xxx` if only affects one of the subsystem

Reply:
There seems to be some confusion between subsystem and area, would it be too long if we prefixed both with area? It seems that many git tools have a limit on the length of the first line (including GitHub).